src/style.css

```
@@ -1 +1,69 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  min-width: 1024px;
+  max-width: 1200px;
+}
+
+body {
+  background-color: #e5e5e5;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  padding: 0 40px 0 50px;
+  align-items: center;
+  justify-content: space-between;
+  background: #fff;
+}
+
+.logo__img {
+  display: block;
+  width: 40px;
+  height: 40px;
+  position: absolute;
+  align-self: center;
+  margin: 10px 0 10px 50px;
+}
+
+.nav__lists {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  text-transform: uppercase;
+}
+
+.nav__link {
+  color: black;
+  position: relative;
+  display: block;
+  font-style: normal;
+  padding: 23px 0;
+  font-weight: 500;
+  line-height: 14px;
+  font-size: 12px;
+  text-transform: uppercase;
+  text-decoration: none;
+  margin-right: 20px;
+}
+
+.nav__item a::after {
```

```suggestion
.is-active::after {
```
plus add position: relative to .is-active